synfig-studio/src/gui/app.cpp

```diff
@@ -3777,6 +3778,22 @@ App::dialog_paragraph(const std::string &title, const std::string &message,std::
 	dialog.add_button(_("OK"),   Gtk::RESPONSE_OK)->set_image_from_icon_name("gtk-ok", Gtk::ICON_SIZE_BUTTON);
 	dialog.set_default_response(Gtk::RESPONSE_OK);
 
+	text_view.signal_key_press_event().connect(( [&](GdkEventKey *ev) {
+
+		SYNFIG_EXCEPTION_GUARD_BEGIN()
+		if (	(ev->type == GDK_KEY_PRESS) &&
+				(ev->keyval == (GDK_KEY_Return) || ev->keyval == (GDK_KEY_KP_Enter) ) &&
+				(ev->state == GDK_CONTROL_MASK) )
+			{
+			dialog.response(-5);
```

Comment:
Please use a named constant here (instead of -5).

Reply:
oh yes definetly xd, was planning on doing so at the end but forgot.